src/TicTac.java

```diff
@@ -121,4 +147,53 @@ public void printBoard() {
             System.out.println();
         }
     }
+
+    public Boolean saveGame(String fileName) {
```

Comment:
notice how similar this whole function is to the corresponding ones for all the other games. I'm thinking you can definitely define something generic you can reuse across all games.

I can help you figure out exactly what API to design, but give it a try yourself first